src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,14 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
```

Comment:
What would be a better place to have this method?

Reply:
I'm not sure how to decide, to be honest. Was choosing between ReadOnlyPerson and Formatter...